week3/about_me_app/config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-
+  get '/about_form', to: 'about_page#form', as: 'form'
+  post '/about_page', to: 'about_page#page', as: 'page'
```

Comment:
Remove the custom route names since it's messing with the tests.
```suggestion
  get '/about_form', to: 'about_page#form'
  post '/about_page', to: 'about_page#page'
```

Reply:
I made the changes, but the test still shows an error.